scripts/build.js

```diff
@@ -40,5 +41,7 @@ compiler.run(function handler (err, stats) {
   process.stdout.write(output);
   process.stdout.write('\n\n');
 
-  ChromeExtensionUtils.pack(manifest);
+  ChromeExtensionUtils.pack(manifest, {
+    zip: isProd
```

Comment:
Ah yes I guess. Well that's not what's written in the `pack` method of `ChromeExtensionUtils`